.travis.yml

```diff
@@ -30,3 +30,17 @@ before_script:
 script:
   - cd test
   - py.test -s -v --durations=10 $TESTS
+
+jobs:
+  include:
+    language:
```

Comment:
Are these blank fields required?

Reply:
Yes, as far as I can see they are required or at least useful since they unset what has been set previously for the none docker build tasks. Here is a test with those lines removed, which shows that Travis-CI goes through the (useless) setup steps which are not required for the Docker test:

https://travis-ci.org/eriktews/OpenWPM/jobs/533198065